applications/registration-react/src/components/app-header/app-header.component.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-
+import { compose } from 'recompose';
```

Comment:
Additional dependency could be avoided as enhancer functions can be combined without it